sema/commons/aggregator/aggregator.py

```
@@ -0,0 +1,40 @@
+        }
+        self.graph = Graph()
+
+    def process(self) -> None:
+        for glb, fmt in self.globs.items():
+            for p in self.input_path.rglob(glb):
+                if p.is_file():
+                    if p.resolve() == self.output_path.resolve():
+                        continue
+                    try:
+                        self.graph.parse(p, format=fmt)
+                    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)